server/src/main/scala/de/frosner/broccoli/nomad/NomadHttpClient.scala

```diff
@@ -127,6 +132,12 @@ class NomadHttpClient(
   val headers: List[(String, String)] =
     sys.env.get(tokenEnvName).map(authToken => ("X-Nomad-Token", authToken)).toList
 
+  /**
+    * Nomad Version. Initiated lazily
+    */
+  override lazy val nomadVersion: String = getNomadVersion()
+    .getOrElse { log.warn("Error fetching nomad version defaulting to 0.4.0"); "0.4.0" }
```

Comment:
That's really old, I'd say 0.8

Reply:
Yes, I will change this. The old tests were running with that version so I kept support for it.